src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './PostList.scss';
+import PropTypes from 'prop-types';
+
+import { Post } from '../Post';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <Post {...post} key={post.id} />
+    ))}
+  </div>
+);
+
+PostList.propTypes = PropTypes.arrayOf({
+  key: PropTypes.number.isRequired,
```

Comment:
Post has `id` property, not `key`. You need to validate props that are passed into the component.